pre-commit-hook.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Define the pre-commit hook file path
+HOOKS_DIR=".git/hooks"
+PRE_COMMIT_FILE="$HOOKS_DIR/pre-commit"
+
+# Create the hooks directory if it doesn't exist
+mkdir -p $HOOKS_DIR
+
+# Write the pre-commit hook script
+cat <<'EOF' > $PRE_COMMIT_FILE
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
@tomaspalma nixos moment :rofl:

Reply:
I learned how to write more portable bash scripts after changing to nixos and noticing no shell script for any repository that I tried was working on it :laughing: